packages/esm-patient-orders-app/src/order-basket-action-button/order-basket-action-button.extension.tsx

```diff
@@ -1,21 +1,42 @@
 import React, { type ComponentProps } from 'react';
 import { useTranslation } from 'react-i18next';
-import { ActionMenuButton, ShoppingCartIcon } from '@openmrs/esm-framework';
+import { ActionMenuButton, ShoppingCartIcon, useWorkspaceContext } from '@openmrs/esm-framework';
 import { useLaunchWorkspaceRequiringVisit, useOrderBasket } from '@openmrs/esm-patient-common-lib';
 
 const OrderBasketActionButton: React.FC = () => {
   const { t } = useTranslation();
   const { orders } = useOrderBasket();
+  const { activeWorkspace } = useWorkspaceContext();
```

Comment:
I don't think this exists? There is a `useWorkspaces()`, but that may require additional details. I don't think it tells you what workspace is active.